javascript/javascript3/week1/giphy.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type= "text/css" href= "gif.css">
+</head>
+<body>
+    <input id="input" type="text" placeholder="search for gif" /> <br><br>
+    <button id="btn">Search GIFs</button><br><br>
+    <input id="limit" type="number" placeholder="number of gifs" /><br><br>
+    
+    
+    <div id = "pic">
```

Comment:
Consider to make a better name for div because the name is not self discriable. for example, imageContainer because the div is a container.